internal/service/eks/access_entry.go

```diff
@@ -82,11 +83,20 @@ func resourceAccessEntry() *schema.Resource {
 			names.AttrTagsAll: tftags.TagsSchemaComputed(),
 			"type": {
 				Type:     schema.TypeString,
-				Computed: true,
+				Optional: true,
+				Default:  "STANDARD",
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"EC2_LINUX",
+					"EC2_WINDOWS",
+					"STANDARD",
+					"FARGATE_LINUX",
+				}, false),
 			},
 			"user_name": {
 				Type:     schema.TypeString,
 				Computed: true,
```

Comment:
Should we remove `Computed: true` for `user_name` as well?

Reply:
`user_name` can be generated by AWS when not specified, for which `Computed: true` should be fine?

Reply:
> If the type of the access entry is Standard, you can specify a username for the access entry. If you don't specify a value for username, Amazon EKS sets one of the following values for you, depending on the type of the access entry and whether the IAM principal that you specified is an IAM role or IAM user. Unless you have a specific reason for specifying your own username, we recommend that don't specify one and let Amazon EKS auto-generate it for you.

from https://docs.aws.amazon.com/eks/latest/userguide/access-entries.html#creating-access-entries